core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.constants.ts

```diff
@@ -16,7 +16,7 @@
  * @fileoverview Constants for the creator dashboard page.
  */
 
-interface TabDetails {
+ interface TabDetails {
```

Comment:
This line should be deindented by one space, as it was before.

Reply:
Done